t04_refactored.py

```diff
@@ -218,9 +218,43 @@ def team_17_adv():
 
 ###################################################################################
 
+
 def team_18_adv():
-    pass
-    # TODO Add your code here
+    """
+    https://docs.google.com/document/d/1-5R7KLwdV7N1FziFAgpwp9FNEqIg8ez1u-0do6Knx3A/edit?usp=sharing
+    Bryce Riley
+    Sam McFarland
+    :return: none
+    """
+
+    is_dead = False
+
+    username = input("What is your name? ")
+    print("Welcome,", username, ", to the woods.")
+    sleep(delay * 3)
+    print("Before you wait too long choose a direction that will let you escape.")
+    print("You can take one of the four directions.")
+
+    direction = input("Which direction in the woods would would you like to take? [North, South, East, West]")
+
+    if direction == "North":
+        print("You are safe, you get back home.")
+        sleep(delay)
+    elif direction == "South":
+        print("You met the cannibal tribe")
+        is_dead = True
```

Comment:
The original prompt asked you to give the user a second chance before they died (e.g., "Pick a number"). Implement that in one of the "death" paths.